gridcomps/Cap/MAPL_Cap.F90

```diff
@@ -288,16 +289,40 @@ subroutine run_model(this, comm, unusable, rc)
       call MPI_COMM_RANK(comm, rank, status)
       _VERIFY(status)
 
+      ! We look to see if the user has set an environment variable for the
+      ! name of the ESMF configuration file. If they have, we use that. If not,
+      ! we use the default of "ESMF.rc" for backward compatibility
+
+      ! Step one: default to ESMF.rc
+
+      esmfConfigFile = 'ESMF.rc'
+      esmfConfigFileLen = len(esmfConfigFile)
+
+      ! Step two: get the length of the environment variable
+      call get_environment_variable('ESMF_CONFIG_FILE', length=esmfConfigFileLen, status=status)
+      ! Step three: if the environment variable exists, get the value of the environment variable
```

Comment:
I'd like to say that this back-and-forth is unneccesary, but I don't immediately see a way to simplify this.